### ISSUE TITLE
The Desktop app does not start 

### ISSUE THREAD

**Issue by Pirognoe** [rancher-sandbox/rancher-desktop#2434]
### Actual Behavior

Hello,
 I have been poking nerdctl and Rancher Desktop as an alternative to Docker desktop and everything was fine - I was able to build image, run it in the container and push it to the Azure registry (via az cli tbh). 
Today when I returned back to the container - the whole app simply do not start up. I have tried resetting everything back to default values and later re-installing the whole thing - nothing helped.

### Steps to Reproduce

Install rancher Desktop
Reload the system
click the Rancher Desktop icon from the Start Menu / Desktop

### Result

failure to start the app

### Expected Behavior

The app starts and the cli commands are working

### Additional Information

I am trying to reproduce this article here https://devblogs.microsoft.com/azure-sql/using-an-azure-container-instance-to-convert-a-bak-to-bacpac-for-import-into-azure-sql-database/

### Rancher Desktop Version

1.4.1

### Rancher Desktop K8s Version

1.24

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 Enterprise 19044.1645

### What CPU architecture are you using?

x64

### Windows User Only

Cisco AnyConnect

**Comment by mook-as** [comment]
Hi!

When you say "do not start up", I assume this means no windows are showing up at all (as opposed to having some kind of error dialog)?

Would you be able to look around in your logs to see if you spot anything? They're in `C:\Users\<you>\AppData\Local\rancher-desktop\logs`; normally there's a button in the UI to show that, but, well…

In particular, `background.log` is the file most likely to have relevant details; however, if you like, posting the whole collection might be the most useful.

**Comment by freakalot** [comment]
I do have the same issue. Here is what the background.log says:
``` 
2022-07-25T07:34:20.850Z: Launching background process Credentials Helper Host Proxy.
2022-07-25T07:34:22.233Z: Stopping background process Win32 socket proxy.
2022-07-25T07:34:22.235Z: mainEvents settings-update: {"version":4,"kubernetes":{"version":"","memoryInGB":2,"numberCPUs":2,"port":6443,"containerEngine":"containerd","checkForExistingKimBuilder":false,"enabled":true,"WSLIntegrations":{},"options":{"traefik":true,"flannel":true},"suppressSudo":false,"experimentalHostResolver":false},"portForwarding":{"includeKubernetesServices":false},"images":{"showAll":true,"namespace":"k8s.io"},"telemetry":true,"updater":true,"debug":false,"pathManagementStrategy":"notset"}
2022-07-25T07:34:22.280Z: Stopping background process Win32 socket proxy.
2022-07-25T07:34:22.344Z: UnhandledRejectionWarning: Error: C:\WINDOWS\system32\wsl.exe exited with code 1
    at ChildProcess.<anonymous> (C:\Users\frd\AppData\Local\Programs\Rancher Desktop\resources\app.asar\dist\app\background.js:17:141990)
    at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12) {
  stdout: '\r\r\n' +
    'Copyright (c) Microsoft Corporation. All rights reserved.\r\r\n' +
    '\r\r\n' +
    'Usage: wsl.exe [Argument]\r\r\n' +
    '\r\r\n' +
    'Arguments:\r\r\n' +
    '\r\r\n' +
    '    --install <Options>\r\r\n' +
    '        Install Windows Subsystem for Linux features. If no options are specified,\r\r\n' +
    '        the recommended features will be installed along with the default distribution.\r\r\n' +
    '\r\r\n' +
    '        To view the default distribution as well as a list of other valid distributions,\r\r\n' +
    "        use 'wsl --list --online'.\r\r\n" +
    '\r\r\n' +
    '        Options:\r\r\n' +
    '            --distribution, -d [Argument]\r\r\n' +
    '                Specifies the distribution to be downloaded and installed by name.\r\r\n' +
    '\r\r\n' +
    '                Arguments:\r\r\n' +
    '                    A valid distribution name (not case sensitive).\r\r\n' +
    '                    \r\r\n' +
    '                Examples:\r\r\n' +
    '                    wsl --install -d Ubuntu\r\r\n' +
    '                    wsl --install --distribution Debian\r\r\n' +
    '\r\r\n' +
    '    --list, -l [Options]\r\r\n' +
    '        Lists distributions.\r\r\n' +
    '\r\r\n' +
    '        Options:\r\r\n' +
    '            --online, -o\r\r\n' +
    "                Displays a list of available distributions for install with 'wsl --install'.\r\r\n" +
    '\r\r\n' +
    '    --help\r\r\n' +
    '        Display usage information.\r\r\n',
  command: [ 'C:\\WINDOWS\\system32\\wsl.exe', '--list', '--quiet' ]
}
```

**Comment by reduardo7** [comment]
This problem happens when WSL2 is not enabled or the Windows Version cannot install it (like Home).
You must enable "WSL2," and it will allow the virtualization to get it working.

**Comment by 3pillardeveloper** [comment]
I have WSL installed but Rancher does not open.

Running Rancher in elevated privileges opens up the program but it cannot install K3S.

### Logs

- I cannot share the contents of the Rancher logs due to an ownership
- I don't appear to own the logs
- I cannot delete them 😠
- I am running Cisco AnyConnect VPN which breaks internet on Windows WSL but I am running [wsl-vpnkit](https://github.com/sakai135/wsl-vpnkit) which unblocks it
- I believe running Rancher in elevated privileges mode caused the log ownership to change (!!!)

![image](https://github.com/rancher-sandbox/rancher-desktop/assets/127785530/60924416-2488-4a84-a8f6-53c10001c037)

**Comment by crucio-009** [comment]
I am seeing the following in the background.log. Can someone guide me what is missing?
`2025-06-27T12:18:08.564Z: Error starting up: Error: Platform "win32" is not supported by RcFilePathManager
    at new b (C:\Users\M K\AppData\Local\Programs\Rancher Desktop\resources\app.asar\dist\app\background.js:1:19466)
    at k (C:\Users\M K\AppData\Local\Programs\Rancher Desktop\resources\app.asar\dist\app\background.js:1:20819)
    at C:\Users\M K\AppData\Local\Programs\Rancher Desktop\resources\app.asar\dist\app\background.js:29:96630
`

**Comment by jandubois** [comment]
The original reporter never responded to the request for logs. Subsequent comments describe distinct environment problems — WSL not installed, and a `wsl-vpnkit` distribution conflict — rather than a single reproducible Rancher Desktop defect.

If Rancher Desktop still fails to start on a system with WSL properly installed, please open a new issue with logs from the current version.
